Source/Core/DolphinWX/Config/AdvancedConfigPane.h

```diff
@@ -5,6 +5,9 @@
 #pragma once
 
 #include <ctime>
+
+#include <wx/dateevt.h>
```

Comment:
These two includes can be replaced with forward declarations as far as I can tell